v2/zis/admin_ui_tutorial_1/src/translations/en.json

```diff
@@ -0,0 +1,8 @@
+{
+  "app": {
```

Comment:
It's probably worth updating these to be somewhat aligned to the actual app description and name

Reply:
@zach-anthony 💇 https://github.com/zendesk/demo_apps/pull/143/commits/ee75d86bce8e0c66465ced5bbf5f98c08ce8c432